install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -8,6 +8,10 @@ global:
   # Default tag for Istio images.
   # Should track latest released version in the branch.
   tag: 0.8.latest
+
+  # git SHA or git tag from istio/proxy repo
+  istioProxyVersion: 0.8.0
```

Comment:
Not sure what that means - 0.8.0 shipped and it's injected, so this will never be a valid combination.

You mean "1.0.0" ? Also the tag above should be 1.0.latest